encrypted-media-respec.html

```diff
@@ -1661,30 +1663,28 @@
                         <dd>
                           <p>Run the following steps:</p>
                           <ol>
-                            <li><p>Clear all stored session data associated with this object, including the <a def-id="sessionId"></a> and license destruction record.</p>
-                              <p class="note">A subsequent call to <a def-id="load"></a> with the value of this object's <a def-id="sessionId"></a> would fail because there is no data stored for that session ID.</p>
-                            </li>
                             <li>
                               <p>
-                                Run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing an empty sequence.
+                                Use <var>cdm</var> to clear all stored session data associated with this object, including the <a def-id="sessionId"></a>
```

Comment:
Yes, and there are also quite a few things in this section which are User Agent procedures (closing the session, updating status, updating expiration, sending a message).

One approach would be to remove all those things and call the "Monitor CDM Changes" algorithm, which would then queue all the appropriate events before the `update()` method promise was resolved.